lib/Screens/Detail.dart.dart

```diff
@@ -0,0 +1,247 @@
+import 'dart:convert';
+import 'package:flutter/material.dart';
+import 'package:flutter_application_1/Screens/newsScreen.dart';
+import 'package:http/http.dart' as http;
+
+class DetailedScreen extends StatefulWidget {
+  final String url;
+  final String text;
+  const DetailedScreen({super.key, required this.url, required this.text});
+
+  @override
+  State<DetailedScreen> createState() => _DetailedScreenState();
+}
+
+class _DetailedScreenState extends State<DetailedScreen> {
+  double height = 50;
+  double width = 50;
+  double height1 = 50;
+  double width1 = 300;
+  Color border_color = Colors.white;
+  Color color = Colors.transparent;
+
+  String? stringResponse;
+  Map? mapResponse;
+  List listResponse = [];
+
+  Future apiCall() async {
```

Comment:
Please write backend's code in a seperate layer as shown during the session. Never right frontend and backend in the same place